.github/workflows/python-publish.yml

```diff
@@ -47,5 +47,5 @@ jobs:
       with:
           upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
           asset_path: dist/*.tar.gz
-          asset_name: manimce-${{ steps.tag.outputs.tag }}.tar.gz
+          asset_name: manim-${{ steps.tag.outputs.tag }}.tar.gz
```

Comment:
I think that this will require an update from @Groctel for the AUR pkgbuild at https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=manimce#n36 -- but if I read the pkgbuild correctly, it will have to be updated for the new release anyways, so not actually a problem.